### ISSUE TITLE
Error attempting to configure project on fresh checkout (xcrun)

### ISSUE THREAD

**Issue by autonomousapps** [square/wire#2368]
We've found a small issue with our desired use of wire-runtime, and so I checked out this repo and was planning to see if I could resolve it myself. I did the following:
```
git clone git@github.com:square/wire.git
./gradlew -p wire-library/ help
```

And got this error:
```bash
> Configure project :wire-runtime-swift
xcrun: error: unable to lookup item 'PlatformPath' from command line tools installation
xcrun: error: unable to lookup item 'PlatformPath' in SDK '/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk'

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':wire-runtime-swift'.
> Failed to query the value of task ':wire-runtime-swift:compileTestSwift' property 'compilerArgs'.
   > Process 'command 'xcrun'' finished with non-zero exit value 1
```
Which I don't understand. I also tried

```
./gradlew -p wire-library/ wire-runtime:help
```
and got the same error.

In case it helps, I also generated a build scan with
```
./gradlew -p wire-library/ help --scan
```
https://scans.gradle.com/s/375uxmebifjae#infrastructure

**Comment by swankjesse** [comment]
It’s attempting an iOS build, which needs Xcode. Does running `xcrun` all by itself help?

**Comment by autonomousapps** [comment]
This is what I get if I just run `xcrun`:
```bash
$ xcrun
Usage: xcrun [options] <tool name> ... arguments ...

Find and execute the named command line tool from the active developer
directory.

The active developer directory can be set using `xcode-select`, or via the
DEVELOPER_DIR environment variable. See the xcrun and xcode-select manual
pages for more information.

Options:
  -h, --help                  show this help message and exit
  --version                   show the xcrun version
  -v, --verbose               show verbose logging output
  --sdk <sdk name>            find the tool for the given SDK name
  --toolchain <name>          find the tool for the given toolchain
  -l, --log                   show commands to be executed (with --run)
  -f, --find                  only find and print the tool path
  -r, --run                   find and execute the tool (the default behavior)
  -n, --no-cache              do not use the lookup cache
  -k, --kill-cache            invalidate all existing cache entries
  --show-sdk-path             show selected SDK install path
  --show-sdk-version          show selected SDK version
  --show-sdk-build-version    show selected SDK build version
  --show-sdk-platform-path    show selected SDK platform path
  --show-sdk-platform-version show selected SDK platform version
```

**Comment by autonomousapps** [comment]
I was able to get my developer environment set up with some gentle nudging from Jesse. Here are the steps for anyone else that runs into this:

1. Install Xcode from https://developer.apple.com/download/all/?q=Xcode
2. Manually move it because it didn't offer the option post-install?! `mv ~/Downloads/Xcode.app/ /Applications/`
3. Select that as my SDK: `sudo xcode-select --switch /Applications/Xcode.app/` (sudo is required!)
4. Validate that xcrun knows about this path now (this is probably optional?): `xcrun --show-sdk-path`
5. And then I could configure the project with `./gradlew -p wire-library/ help`

Closing the issue as resolved, but it might be nice to have this in the README somewhere.
